sdk/src/types/block/address/implicit_account_creation.rs

```diff
@@ -26,6 +29,12 @@ impl ImplicitAccountCreationAddress {
     }
 }
 
+impl StorageScore for ImplicitAccountCreationAddress {
+    fn storage_score(&self, params: StorageScoreParameters) -> u64 {
+        params.implicit_account_creation_address_offset() + self.0.storage_score(params)
```

Comment:
Can we move `self.0.storage_score(params)` to the offset definition? Closer to the TIP

Reply:
I can't even find this TIP

Reply:
https://github.com/iotaledger/tips/blob/tip42/tips/TIP-0042/tip-0042.md#storage-score-2